src/internal/components/dropdown/styles.scss

```diff
@@ -70,6 +70,11 @@
   &.occupy-entire-width {
     min-width: 100%;
   }
+  &.stretch-beyond-trigger-width {
+    width: max-content;
+    // This maps to the XXS container breakpoint as defined in src/internal/breakpoints.ts
+    max-width: 465px;
```

Comment:
Can we avoid this duplication by setting this value with inline styles from JS?

Reply:
Good idea, let's do that.